invenio_queues/version.py

```diff
@@ -12,4 +12,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = '1.0.0a3'
+__version__ = '1.0.0a4'
```

Comment:
Shouldn't we make a `1.0.0` release? Or are there a lot of things missing from the module? (trying to find a checklist or similar...)

Reply:
release checklist is here <https://github.com/inveniosoftware/invenio-queues/issues/12>

Reply:
I think the minimum requirements were: having docs and module being prod ready

Reply:
There are docs, but apparently they don't seem to be building correctly on Read the docs: https://invenio-queues.readthedocs.io/en/latest/. I'm checking on the setting to see if it's easy to fix